geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-datastore-common/src/main/scala/org/locationtech/geomesa/kafka/LiveFeatureCacheGuava.scala

```diff
@@ -88,5 +125,7 @@ class LiveFeatureCacheGuava(override val sft: SimpleFeatureType,
 
   override def getFeatureById(id: String): FeatureHolder = cache.getIfPresent(id)
 
+  override def close(): Unit = consistencyChecker.foreach(_.shutdown())
```

Comment:
shouldn't shutdown be moved to after line 82 and this should force a shutdown? since shutdown only stops it from taking new work.

Reply:
hmm, i assumed it would stop it from re-scheduling after the last run. I will double-check that

Reply:
cool.